### PR TITLE
Reorder branching procedure

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

File: procedures/foreman/branch.md.erb [hunk 1]
```diff
@@ -21,25 +59,8 @@
 * [ ] Clone Debian nightly repos to <%= release %> using [copy/freight instructions](https://projects.theforeman.org/projects/foreman/wiki/Debian_Packaging#Project-sources)
 * [ ] Edit [foreman-packaging's PR template](https://github.com/theforeman/foreman-packaging/blob/master/.github/PULL_REQUEST_TEMPLATE.md) to add <%= release %> and remove the old release
 
-# Main code repos
+# Bracnh main code repos
```
Comment (ekohl) [inline procedures/foreman/branch.md.erb]:
```suggestion
# Branch main code repos
```

**Comment by ehelms** [comment]
@tbrisker Needs commenting and rebasing

**Comment by tbrisker** [comment]
Updated with n-2, n-1 weeks suggestion and rebased

**Comment by tbrisker** [comment]
@ekohl if this is fine with you I'd like to use this for 1.23 already.

File: procedures/foreman/branch.md.erb [hunk 2]
```diff
@@ -1,16 +1,45 @@
...
+
+* [ ] Announce start of stabilization week to discourse [development category](https://community.theforeman.org/c/development).
+* Prepare the manual for the new version:
+  * [ ] Change `$latest` and `$next` parameters on [web class](https://github.com/theforeman/foreman-infra/blob/master/puppet/modules/web/manifests/init.pp) to point to the new version numbers
+  * [ ] Copy [website manual content](https://github.com/theforeman/theforeman.org/tree/gh-pages/manuals) from nightly to <%= release %> and update version numbers mentioned in it.
+  * [ ] Clean up deprecation and upgrade warnings from nightly manual.
 * [ ] Refresh unattended templates
   * [ ] Sync the community-templates to Foreman core, by running `script/sync_templates.sh`
   * [ ] Create a <%= @release %>-stable branch in [community-templates](https://github.com/theforeman/community-templates)
+* [ ] Update puppet module versions in installer to the newly released versions
```
Comment (ekohl) [inline procedures/foreman/branch.md.erb]: We have nightly builds so they should automatically be pulled in. During branching there's already a task to pin to released versions which automatically takes the latest ones.

Comment (tbrisker) [reply procedures/foreman/branch.md.erb]: removed

**Pull request merged**